### PR TITLE
Fix readonly parameters margins

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Also fixes next button title going back to the plan page of the update service wizard.

### CODE REVIEW

File: dist/less/catalogParameters.less [hunk 1]
```diff
@@ -8,6 +8,19 @@
     padding-left: 0;
   }
   &.readonly {
+    .control-label {
+      padding-right: 0;
+      word-break: break-word;
```
Comment (spadgett) [inline dist/less/catalogParameters.less]: I don't think this is a standard `word-break` value. Do you mean `word-wrap: break-word;`?

https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

**Comment by spadgett** [comment]
@jeff-phillips-18 Do you mind pasting a screenshot?

File: dist/less/catalogParameters.less (hunk 1 above)
Comment (jeff-phillips-18) [reply dist/less/catalogParameters.less]: Yes, sorry.

**Comment by jeff-phillips-18** [comment]
image

**Comment by spadgett** [comment]
I feel like the parameters need some top margin. It seems crowded.

Should we reuse the same font-style as the overview for the "PARAMETERS" header? It looks a bit bigger here.

The left indentation looks strange to me still.

@jwforres @beanh66 opinion?

**Comment by jwforres** [comment]
+1 to needing some top margin
+1 to indention looking strange

**Comment by beanh66** [comment]
@spadgett After seeing this, agreed all around. 

+1 to adding more space above PARAMETERS
+1 to left aligning, I think we will be fine without the indent since we have the PARAMETERS header
+1 the header should be the same font we use on the overview for things like DEPLOYMENT

**Comment by jeff-phillips-18** [comment]
Those changes:
image

**Comment by jeff-phillips-18** [comment]
For what it's worth, that styling is all in console and will not be part of this PR.

**Comment by serenamarie125** [comment]
PARAMETERS looks to be a smaller font than Reveal Values now @spadgett @jeff-phillips-18

**Comment by jeff-phillips-18** [comment]
Yes, 11px vs 13px

**Comment by jeff-phillips-18** [comment]
A few more tweaks (again not in this PR):
image

**Comment by serenamarie125** [comment]
not as noticeable now that you've changed the spacing.  Looks better to me!

**Pull request merged**